pkg/apis/scheduling/types.go

```diff
@@ -57,7 +57,20 @@ const (
 type PodGroupConditionType string
 
 const (
+	// PodGroupUnschedulableType is Unschedulable event type
 	PodGroupUnschedulableType PodGroupConditionType = "Unschedulable"
+
+	// PodGroupScheduled is scheduled event type
+	PodGroupScheduled PodGroupConditionType = "Scheduled"
```

Comment:
Add to both v1alpha1 and v1alpha2 types.go？

Reply:
done